src/main/java/org/localhost/pizzeria/messaging/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,81 @@
+package org.localhost.pizzeria.messaging.config;
+
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RabbitMQConfig {
+
+    @Value("${rabbitmq.queue.supply-check}")
+    private String queueName;
+
+    @Value("${rabbitmq.exchange.supply-check}")
+    private String exchangeName;
+
+    @Value("${rabbitmq.exchange.supply-check-dlq-exchange}")
+    private String dlqExchangeName;
+
+    @Value("${rabbitmq.routing-key.supply-check}")
+    private String routingKey;
+
+    @Value("${rabbitmq.queue.supply-check-dlq}")
+    private String queueDlq;
+
+    @Bean
+    TopicExchange exchange() {
+        return new TopicExchange(exchangeName);
+    }
+
+    @Bean
+    FanoutExchange deadLetterExchange() {
+        return new FanoutExchange(dlqExchangeName);
+    }
+
+    @Bean
+    Queue deadLetterQueue() {
```

Comment:
ja bym nie tworzyl tego przez springa - zrob sobie skrypt do konfiguracji rabbita
1. Jezeli robisz to przez springa jestes uzalezniony od wdrozen
2. Jestes uzalezniony od tego jaka konfiguracje dopuszcza spring - moze byc tak ze ci ona nie wystarczy
3. Przy kazdym bledzie konfiguracji rabbita cala twoja aplikacja lezy 
4. Aplikacja powinna byc niezalezna od konfiguracji srodowiska